gravitee-apim-console-webui/src/management/api/endpoints-v4/backend-services/endpoint-group/general/api-endpoint-group-general.component.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component, Input } from '@angular/core';
+import { FormGroup } from '@angular/forms';
+
+import { ProxyGroupLoadBalancerEnum } from '../../../../../../entities/proxy';
+
+@Component({
+  selector: 'api-endpoint-group-general',
+  template: require('./api-endpoint-group-general.component.html'),
+  styles: [require('./api-endpoint-group-general.component.scss')],
+})
+export class ApiEndpointGroupGeneralComponent {
+  @Input() generalForm: FormGroup;
```

Comment:
```suggestion
  @Input() formGroup: FormGroup;
```

to be consistent with what we have in other component (https://github.com/gravitee-io/gravitee-api-management/blob/0c28d0a471af14d9878bda8c8ef2b70e5feae126/gravitee-apim-console-webui/src/management/api/entrypoints-v4/edit/api-entrypoints-v4-add-dialog.component.ts#L48)

Reply:
Actually the formGroup is there - it's called endpointGroupForm - I'll rename that. This still makes sense to keep it generalForm as there will be other tabs within the formGroup if you see https://github.com/gravitee-io/gravitee-api-management/blob/5a81f661017d2142445c303f2da5f37d77400054/gravitee-apim-console-webui/src/management/api/proxy/endpoints/groups/edit/api-proxy-group-edit.component.html#L28